### ISSUE TITLE
💥 Accessibility - Cannot Increase Font Size in App on iOS 26.6

### ISSUE THREAD

**Issue by persistensk** [super-productivity/super-productivity#9424]
### Steps to Reproduce

1. Tested on iPhone 16 Pro Max. 
2. Ensure iOS is updated to latest version (currently, 26.6 - Note: your “Meta info”  below says “ iPhone OS 18_7” but that is incorrect, as I have iOS 26.6). 
3. Install the latest version of the Super Productivity app (currently,18.16.0) from Apple App Store, and Open the app. 
4. After the initial steps, start entering tasks. 
5. Font looks very small. 
6. No setting to increase font size in app. 
7. Go to iOS Settings > Accessibility > Vision > Display and Text Size > Larger Text > switch setting to On > move slider on bottom of screen to the right > Dynamic Type sample text size increases. 
8. Go back to the Super Productivity app.
9. Text size has not changed, still very small.
10. Expected result: Text size in app should increase.
11.  Swipe down from the top right corner to open the iOS Control Center.
12. Long press on an empty part of the Control Center screen.
13. Tap on Add a Control. 
14. Search for Text Size widget and add it.
15. Short tap on an empty part of the control centre screen to go back to the normal Control Centre display.
16. Find the Text Size widget and swipe upward on it to increase the text size. 
17. Go back to the Super Productivity app.
18. The text size has not changed, it is still very small.
19. Expected result: the text size in the app should be larger.

Note: being able to enlarge the text size is very important for accessibility for people with visual disabilities, as well as anyone with tired eyes.

Thank you in advance for fixing this.

### Additional Console Output

N/A. 

### URL
capacitor://localhost/#/tag/TODAY/tasks

### Meta Info
META: SP18.16.0I __ Browser – en-CA – iPhone – en-CA – UA:Mozilla/5.0 (iPhone; CPU iPhone OS 18_7 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148

### Actions Before Error
```

```

**Comment by persistensk** [comment]
This is understandable. Thank you for your work on this app, which generally looks very good to me, except for the issue above.
